src/cli/cli_tcp.cpp

```diff
@@ -290,6 +341,34 @@ template <> otError TcpExample::Process<Cmd("bind")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli tcp connect
+ * @code
+ * tcp connect fe80:0:0:0:a8df:580a:860:ffa4 30000
+ * Done
+ * TCP: Connection established
+ * @endcode
+ * @code
+ * tcp connect 172.17.0.1 1234
+ * Connecting to synthesized IPv6 address: fdde:ad00:beef:2:0:0:ac11:1
+ * Done
+ * @endcode
+ * @cparam tcp connect @ca{ip} @ca{port} [@ca{fastopen}]
+ * Parameter:
```

Comment:
This doesn't add any value, since the header of Parameters is right before it on the web site.  Let's use a proper lead in sentence or just remove this and start with the list